__tests__/sources.ts

```diff
@@ -1859,6 +1859,22 @@ describe('mutation createSquad', () => {
     );
   });
 
+  it('Should throw error when post moderation is required, but posting is restricted to moderators only', async () => {
```

Comment:
we usually use lowercase to keep it readable in outputs
```suggestion
  it('should throw error when post moderation is required, but posting is restricted to moderators only', async () => {
```